setup.py

```diff
@@ -435,4 +435,11 @@ def do_setup():
 
 
 if __name__ == "__main__":
+    # Warn about py2 support going away. This likely won't show up if installed
```

Comment:
Maybe we should add similar warning in the airflow script. Then it will be printed every time you run any cli command. It's a bit spammy, but maybe it's the best way to warn users.

Reply:
Hmmm maybe -- that is probably a bit _too_ spammy

Reply:
Maybe just once after installation or at some specific operations - for example when running webserver or scheduler ? That might be less spammy but still run often by the users/devops maintaining Airflow.

Reply:
Added it to webserver, scheduler, version, upgradedb, initdb, and resetdb commands

I also re-used coloring code from pip if it's available.